src/DynamoInstall/Product.wxs

```diff
@@ -69,7 +69,7 @@
         <Directory Id="ADESK_DATA" Name="Autodesk">
         </Directory>
         <Directory Id="DYNAMO_PROGDATA" Name="Dynamo">
-          <Directory Id="PROGDATA_07" Name="0.7"/>
+          <Directory Id="PROGDATA_08" Name="0.8"/>
```

Comment:
Do we need to have the version number in id? Better call in PROGDATA, so that you don't need to update the wixproj when you change the version number next time.